board/e5_dev_board/Kconfig

```diff
@@ -0,0 +1,15 @@
+# Copyright (c) 2021 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+#
+# This file is the application Kconfig entry point. All application Kconfig
+# options can be defined here or included via other application Kconfig files.
+# You can browse these options using the west targets menuconfig (terminal) or
+# guiconfig (GUI).
```

Comment:
if the file was modified change this as well

Reply:
Have you researched the topic of changing license? I know, that Apache-2.0 provides the code AS IS, but I don't have broad knowledge about licensing. I just want to be safe in this regard, there is nothing wrong with using open source code in the product.